java/style-wms-layer/README.md

```diff
@@ -1,18 +1,33 @@
-# Style WMS Layer
-Shows how to change the style of a WMS layer.
+# Style WMS layer
 
-![Style WMS Layer App](style-wms-layer.png)
+Change the style of a Web Map Service (WMS) layer.
+
+![Image of style WMS layers](style-wms-layer.png)
+
+## Use case
+
+Layers hosted on WMS may have different pre-set styles available to apply to them. Swapping between these styles can help during visual examination of the data. For example, increasing the contrast of satellite images can help in identifying urban and agricultural areas within forested areas.
 
 ## How to use the sample
-Click the toggle button at the bottom of the screen to switch between styles.
+
+Once the layer loads, the toggle button will be enabled. Tap it to toggle between the first and second styles of the WMS layer.
```

Comment:
Nit picky, but I would remove the first instance of 'toggle' to describe the button as it doesn't look like a toggle, it just looks like a normal button.
```suggestion
Once the layer loads, the button will be enabled. Tap it to toggle between the first and second styles of the WMS layer.
```